README.rst

```diff
@@ -46,6 +46,56 @@ corner lets you edit the following information:
 
 * Access control: to save changes, you need to grant the IDE permission to write
   to your GitHub repo or, if you started with a D-Wave example, create a fork.
-* Environmental variables: the IDE automatically sets your SAPI token (token used
+* Environmental variables such as your SAPI token (token used
   to authenticate client sessions when you connect to remote solvers such as D-Wave
   quantum computers and hybrid solvers).
+
+Configuration
+-------------
+
+Some more advanced configurations include the following:
+
+Repository Requirements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create a YAML file to open a repository with your required packages
+loaded as follows::
+
+  tasks:
+   - init: pip install -r requirements.txt
+
+User Dockerfile
+~~~~~~~~~~~~~~~
+
+The IDE uses a Docker image, based on python:3.7-slim, available within
+the IDE as ``dwavesys/leapide:latest``. To build on top of it, your Dockerfile
+should look like this::
+
+  FROM dwavesys/leapide:latest
+
+  USER root
+
+  # install system packages
+  RUN apt-get update \
+      && apt-get install -yq --no-install-recommends \
+          clang \
+          libboost-dev \
+      && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/*
+
+  # install system-wide python package
+  RUN pip install --no-cache-dir \
+          dwave-ocean-sdk==4.0.0 \
+      && rm -rf /tmp/*
+
+API token
```

Comment:
Maybe mention that this is valid for new users only (registered after Dec 2).

Old users can get this feature by 1) deleting `DWAVE_API_TOKEN` env var and 2) creating a new workspace.

Reply:
I'm thinking this info is more relevant to release notes. If we thought large numbers of users might be affected that would justify transition info in the tool's documentation but I don't believe that's the case.